iptiny : add support vxlan configuration

ip-tiny is not supporting some of the options needed for vxlan configuration.

--- a/ip/Makefile
+++ b/ip/Makefile
@@ -19,7 +19,7 @@ include ../config.mk
 
 STATIC_SYM_FILTER:=
 ifeq ($(IP_CONFIG_TINY),y)
-  STATIC_SYM_FILTER:=iplink_can.c iplink_ipoib.c iplink_vxlan.c
+  STATIC_SYM_FILTER:=iplink_can.c iplink_ipoib.c
   CFLAGS += -DIPROUTE2_TINY
 endif
 STATIC_SYM_SOURCES:=$(filter-out $(STATIC_SYM_FILTER),$(wildcard *.c))
